project.clj

```diff
@@ -4,33 +4,37 @@
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
 
-  :profiles {:dev {:dependencies [[org.clojure/clojure "1.7.0"]
-                                  [org.clojure/clojurescript "0.0-2760"]
-                                  [org.clojure/tools.nrepl "0.2.5"]
+  :profiles {:dev {:dependencies [[org.clojure/clojure "1.8.0"]
```

Comment:
Does this mean it no longer works on 1.7? (there was already a profile for 1.8).

Reply:
never mind, I missed your last comment